src/util/nettest.js

```diff
@@ -48,3 +48,107 @@ export const getDeckIcon = (iconName, other) => {
     <i className={className} />
   )
 }
+
+export const MLAB_LOC_TO_COUNTRY = {
```

Comment:
A more scalable way to get this job done could be:

- import all the IATA codes of all airport

- match just the first three letters to get the country

This would be robust to mlab expanding further.